fuse/dl/README.md

```diff
@@ -0,0 +1,75 @@
+# FuseMedML DL package
+
+The fuse.dl package contains core DL related modules that make heavy use of existing DL libraries. Currently, PyTorch and PyTorch Lightning are the two primary frameworks that FuseMedML is based on.
+
+## lightning
+FuseMedML uses PyTorch Lightning as it's model training "manager".  
+There are two main supported ways in which lightning is used in fuse:
+### 1. Ready-made wrapper use (suitable mostly for supervised learning use-cases)
+(see [MNIST example](../../examples/fuse_examples/imaging/classification/mnist/run_mnist.py)).  
+In this mode, we use `LightningModuleDefault`, a FuseMedML class that inherits from PyTorch Lightning's `LightningModule` class. It is instantiated as follows:
+
+```
+pl_module = LightningModuleDefault(model_dir=model_dir, 
+                                       model=model,
+                                       losses=losses,
+                                       train_metrics=train_metrics,
+                                       validation_metrics=validation_metrics,
+                                       best_epoch_source=best_epoch_source,
+                                       optimizers_and_lr_schs=optimizers_and_lr_schs)
+```
+where the user is only responsible to supply the required arguments, including the model, losses, train and validation metrics, all of which are in fuse style (classes implemented or supported in FuseMedML).
+
+    
+### 2. Custom use (suitable for any use case)
+(see [MNIST example](../../examples/fuse_examples/imaging/classification/mnist/run_mnist_custom_pl_imp.py))  
+
+In this more flexible and custom mode, the user is responsible to implement their own `LightningModule` tailored to their project, with all standard methods required by PyTorch Lightning such as `forward`, `training_step` etc'.  
+Once such a custom module exists, it is instantiated with a user defined set of arguments:
+```
+pl_module = CustomLightningModule(**custom_args)
+```  
+
+Note: previously, a dedicated PyTorch based [manager](managers) was used which is now deprecated. Some of the implemented [examples](../../examples) still use it but we plan to port them to use the lightning manager.
+
```

Comment:
Can you mention als pl_funcs a collection of functions that can help to implement CustomLightningModule